src/components/Collapse.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import { Collapse as GrafanaCollapse } from '@grafana/ui';
+import { css } from 'emotion';
+
+interface Props {
+  isOpen?: boolean;
+  label: string;
+  loading?: boolean;
+  collapsible?: boolean;
+  onToggle?: (isOpen: boolean) => void;
+}
+
+const containerStyles = css`
+  .panel-container {
+    border-right: none;
+    border-left: none;
+    border-bottom: none;
+    margin-bottom: 0;
+  }
+  .panel-container > div:first-of-type {
+    padding: 16px 0px;
+  }
+  .panel-container > div:nth-child(2) {
+    padding: 0 8px;
```

Comment:
Might be worthwhile considering using the grafana theme for various hard-coded pixel values. It has design tokens for e.g. spacing and typography.